api_app/core/views.py

```diff
@@ -49,20 +52,16 @@ def perform_retry(self, report: AbstractReport):
         runtime_configuration = {report.name: report.runtime_configuration}
         return plugins_to_execute, runtime_configuration
 
+    @add_docs(
+        description="Kill running plugin by closing celery task and marking as killed",
+        request=None,
+        responses={
+            204: None,
+            400: None,
```

Comment:
Won't be `400: None`, if `APIException` is raised, it is automatically sent in response.

Reply:
Just remembered, I had tried adding the response for it, but it wasn't showing up in the schema, will retry.

Reply:
You can leave it for now. This is something that needs to be added universally to all API endpoints. So there may be a better way to do so.

Reply:
Just removing the `400: None` should suffice for now.